tests/apps/qchem/test_dynamics.py

```diff
@@ -326,3 +326,83 @@ def test_negative_state(self, e):
         with pytest.raises(ValueError, match="The excited state must not contain negative values"):
             samples, state, _ = e
             dynamics.prob(samples, [-i for i in state])
+
+
+class TestMarginal:
+    """Tests for the function ``strawberryfields.apps.qchem.dynamics.marginal``"""
+
+    mu = np.array([0.00000000, 2.82842712, 0.00000000, 0.00000000, 0.00000000, 0.00000000])
+    V = np.array(
+        [
+            [1.0, 0.0, 0.0, 0.0, 0.0, 0.0],
+            [0.0, 1.0, 0.0, 0.0, 0.0, 0.0],
+            [0.0, 0.0, 1.0, 0.0, 0.0, 0.0],
+            [0.0, 0.0, 0.0, 1.0, 0.0, 0.0],
+            [0.0, 0.0, 0.0, 0.0, 1.0, 0.0],
+            [0.0, 0.0, 0.0, 0.0, 0.0, 1.0],
+        ]
+    )
+    n = 10
+    p = np.array(
```

Comment:
This is the most vertical code I've ever seen! 
I get that it's probably the result of using black so we should maybe keep as is, but I find it very inelegant

Reply:
It is much shorter now.